sites/all/themes/orwell/css/orwell.main.css

```diff
@@ -3901,7 +3901,13 @@ html.js input.form-autocomplete {
   text-decoration: none;
 }
 
-.page-user .library-info a {
+.page-user .pane-content {
+  font-size: 1rem;
+  line-height: 1.8rem;
```

Comment:
Line height should not use a unit. Percent is cool tho.

Reply:
But that sets the line-height relative to the elements font-size, not an absolute value?